res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -104,7 +104,14 @@ DJ2GO2Touch.Deck = function(deckNumbers, midiChannel) {
         this.hotcueButtons[i] = new components.HotcueButton({
             group: "[Channel" + script.deckFromGroup(this.currentDeck) + "]",
             midi: [0x94 + midiChannel, 0x00 + i],
-            number: i
+            number: i,
+            input: function(channel, control, value, status, _group) {
```

Comment:
If the received control is smaller than or equal to 8, send _activate to hotcue.
If the received control is greater than 8, send _clear to hotcue.